bundle/artifacts/whl/from_libraries.go

```diff
@@ -1,79 +0,0 @@
-package whl
-
-import (
-	"context"
-	"path/filepath"
-
-	"github.com/databricks/cli/bundle"
-	"github.com/databricks/cli/bundle/config"
-	"github.com/databricks/cli/bundle/libraries"
-	"github.com/databricks/cli/libs/diag"
-	"github.com/databricks/cli/libs/log"
-)
-
-type fromLibraries struct{}
-
-func DefineArtifactsFromLibraries() bundle.Mutator {
```

Comment:
Having seen the rest of the PR, I now see that the files under artifacts are included in the upload as well.

It's a bit odd to have this cross-over between the artifacts and libraries packages but see how it is necessary. The alternative is artificially creating an artifact entry for every library we see that's unaccounted for, which is also backwards.

Reply:
>  The alternative is artificially creating an artifact entry for every library we see that's unaccounted for,

I think this would be more confusing. The way I reason about it right now is: what we do is collect all references to libraries in any part of bundle configuration, today they are task libraries, environments and artifact files. And then we upload them all. If later we add more library-like things, we would add it the same way to this new mutator

Reply:
It also makes artifact use more clear - you only need to use them if you want to build something as part of DABs deploy, just for uploads you can reference your local libs in configuration

Reply:
I like that explanation, thanks.